airflow/models.py

```diff
@@ -756,8 +740,77 @@ def command(
         the orchestrator.
         """
         dag = self.task.dag
-        iso = self.execution_date.isoformat()
-        cmd = "airflow run {self.dag_id} {self.task_id} {iso} "
+
+        # Keeping existing logic, but not entirely sure why this is here.
+        if not pickle_id and dag:
+            if dag.full_filepath != dag.filepath:
+                path = "DAGS_FOLDER/{}".format(dag.filepath)
```

Comment:
I'm late to the party, but I think there should be a [`path = None` initialization above](https://github.com/mtdewulf/incubator-airflow/commit/6d878698f9f86649f448bb89d306b54a6aeee294)?